src/dom-to-react.ts

```diff
@@ -37,10 +37,9 @@ export default function domToReact(
   const { cloneElement, createElement, isValidElement } =
     options?.library || React;
 
-  let index = 0;
   const nodesLength = nodes.length;
 
-  for (; index < nodesLength; index++) {
+  for (let index = 0; index < nodesLength; index++) {
```

Comment:
How about adding `let index = 0` inside the for statement?

Reply:
Works